doc/cordova.txt

```diff
@@ -43,4 +49,7 @@ Examples
     cordova-cli requirements android    
     cordova-cli build android --verbose
     cordova-cli run android
-    cordova-cli build android --release -- --keystore="..\android.keystore" --storePassword=android --alias=mykey
\ No newline at end of file
+    cordova-cli build android --release -- --keystore="..\android.keystore" --storePassword=android --alias=mykey
+    cordova-cli telemetry on
+    cordova-cli telemetry off
+    cordova-cli build --no-telemetry
```

Comment:
These examples are not necessarily needed here as they are simple and not common scenarios - it will be nice to keep this output as small as it could be.